uct/prior.c

```diff
@@ -113,13 +113,87 @@ print_dcnn_best_moves(struct tree_node *node, struct prior_map *map,
         fprintf(stderr, "]\n");	
 }
 
+static float*
+dcnn2_get_moves_data(struct board *b, enum stone color)
```

Comment:
I'm not that fond of moving this to uct/prior.c, but I assume you moved it out of dcnn.cpp due to the C++ issues that caused the segfaults?

Maybe we can later change it to dcnn.c + dcnn-caffe.cpp, but there's no rush for that. It's fine here imho for the time being if the above is the reason.